### PR TITLE
feat: add github ci step

### PR DESCRIPTION
**Pull request by CoderTH**
Add a new ut step to the CI pipeline

image

### CODE REVIEW

File: .github/workflows/golang-build.yaml [hunk 1]
```diff
@@ -36,6 +36,17 @@ jobs:
...
+      name: Unit test
+      runs-on: ubuntu-latest
+      steps:
+        - name: Checkout code
+          uses: actions/checkout@v4
+        - name: Install Go
+          uses: actions/setup-go@v4
+          with:
+            go-version: '1.20'
+        - run: make test
```
Comment (elezar) [inline .github/workflows/golang-build.yaml:49]: Just a note is that the the `build.Tests` step also runs the unit tests, although these are not as visible as your job.

Comment (CoderTH) [reply .github/workflows/golang-build.yaml:49]: So can I assume that Build.Test doesn't really need it? I deleted it.

**Comment by CoderTH** [comment]
> @CoderTH please sign-off your commit so that I can retrigger the checks.

ok,done

**Comment by CoderTH** [comment]
@elezar cc

**Review by elezar — APPROVED** [review]
LGTM

**Pull request merged**